apps/server/src/infra/sync/tsp/tsp-sync.strategy.spec.ts

```diff
@@ -162,6 +187,32 @@ describe(TspSyncStrategy.name, () => {
 
 				expect(provisioningService.provisionData).toHaveBeenCalledWith(oauthDataDto);
 			});
+
+			describe('when feature tsp migration is enabled', () => {
+				it('should fetch teacher migrations', async () => {
+					setup();
+
+					await sut.sync();
+
+					expect(tspSyncService.fetchTspTeacherMigrations).toHaveBeenCalled();
+				});
+
+				it('should fetch student migrations', async () => {
+					setup();
+
+					await sut.sync();
+
+					expect(tspSyncService.fetchTspStudentMigrations).toHaveBeenCalled();
+				});
+
+				it('find user by tsp Uid', async () => {
+					setup();
+
+					await sut.sync();
+
+					expect(tspSyncService.findUserByTspUid).toHaveBeenCalled();
+				});
+			});
```

Comment:
Is this duplicated code? Because a little below is similar code.

Reply:
Clean up done